tensorflow_datasets/core/download/downloader.py

```diff
@@ -148,21 +149,39 @@ def _sync_download(self, url, destination_path):
         'https': os.environ.get('TFDS_HTTPS_PROXY', None),
         'ftp': os.environ.get('TFDS_FTP_PROXY', None)
     }
+    CA_BUNDLE = os.environ.get(
```

Comment:
local variables should not be written with ALL_CAPS

Reply:
Sure. :)